Odata-docs/odatalib/v7/edm/iedmtargetpath-for-annotations-target.md

```diff
@@ -0,0 +1,463 @@
+        <Property Name="Name" Type="Edm.String" Nullable="false" />
+        <Property Name="Address" Type="NS.Address" Nullable="false" />
+        <NavigationProperty Name="Cities" Type="Collection(NS.City)" />
+      </EntityType>
+      <EntityType Name="VipCustomer" BaseType="NS.Customer" />
+      <EntityType Name="City">
+        <Property Name="Name" Type="Edm.String" Nullable="false" />
+      </EntityType>
+      <Term Name="MyTerm" Type="Edm.String" />
+      <EntityContainer Name="Default">
+        <EntitySet Name="Customers" EntityType="NS.Customer" />
+        <Singleton Name="Me" Type="NS.Customer" />
+      </EntityContainer>
+      <Annotations Target="NS.Default/Customers/Name">
+        <Annotation Term="NS.MyTerm" String="Name OutOfLine MyTerm Value" />
+      </Annotations>
+    </Schema>
+  </edmx:DataServices>
+</edmx:Edmx>
+```
+
+## Read out-of-line annotation
+Replace the contents of _Program.cs_ file with the following code:
+
+```csharp
+using Microsoft.OData.Edm.Csdl;
+using Microsoft.OData.Edm.Vocabularies;
+using Microsoft.OData.Edm;
+using System.Reflection;
+using System.ComponentModel;
+using Microsoft.OData.Edm.Validation;
+using System.Xml;
+using static System.Runtime.InteropServices.JavaScript.JSType;
+using System.Xml.Linq;
+
+namespace AnnotationsTarget
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            // Read Annotations
+            string csdl = @"<?xml version=""1.0"" encoding=""UTF-8""?>
+<edmx:Edmx xmlns:edmx=""http://docs.oasis-open.org/odata/ns/edmx"" Version=""4.0"">
+	<edmx:DataServices>
+		<Schema xmlns=""http://docs.oasis-open.org/odata/ns/edm"" Namespace=""NS"">
+			<ComplexType Name=""Address"">
+				<Property Name=""Road"" Type=""Edm.String"" Nullable=""false"" />
+				<NavigationProperty Name=""City"" Type=""NS.City"" Nullable=""false"" />
+			</ComplexType>
+			<ComplexType Name=""DerivedAddress"" BaseType=""NS.Address"">
+				<Property Name=""MyRoad"" Type=""Edm.String"" Nullable=""false"" />
+			</ComplexType>
+			<EntityType Name=""Customer"">
+				<Key>
+					<PropertyRef Name=""Id""/>
+				</Key>
+				<Property Name=""Id"" Type=""Edm.Int32"" Nullable=""false"" />
+				<Property Name=""Name"" Type=""Edm.String"" Nullable=""false"" />
+				<Property Name=""Address"" Type=""NS.Address"" Nullable=""false"" />
+				<NavigationProperty Name=""Cities"" Type=""Collection(NS.City)"" />
+			</EntityType>
+			<EntityType Name=""VipCustomer"" BaseType=""NS.Customer"" />
+			<EntityType Name=""City"">
+				<Property Name=""Name"" Type=""Edm.String"" Nullable=""false"" />
+			</EntityType>
+			<Term Name=""MyTerm"" Type=""Edm.String"" />
+			<EntityContainer Name=""Default"">
+				<EntitySet Name=""Customers"" EntityType=""NS.Customer"" />
+				<Singleton Name=""Me"" Type=""NS.Customer"" />
+			</EntityContainer>
+			<Annotations Target=""NS.Default/Customers/Name"">
+				<Annotation Term=""NS.MyTerm"" String=""Name OutOfLine MyTerm Value"" />
```

Comment:
'Name OutOfLine MyTerm Value' for test codes is ok. but, in my opinion, as a 'doc' or 'tutorial', at least we should use 'meaningful' string for the value, or at least 'readable' string.